docs/source/community/contribution_guide.rst

```diff
@@ -2,18 +2,19 @@ PyTorch Contribution Guide
 ==========================
 
 PyTorch is a GPU-accelerated Python tensor computation package for
-building deep neural networks built on tape-based autograd systems.
+building deep neural networks based on tape-based autograd systems.
```

Comment:
based twice? maybe "using a"

Reply:
I wrote the original copy, so looks good! @svekars can you change?

Reply:
Addressed.